test/v1beta1/occurrence_test.go

```diff
@@ -201,6 +214,36 @@ func TestOccurrence(t *testing.T) {
 						secondVulnerabilityOccurrence,
 					},
 				},
+				{
+					name:   "match resourceUri startsWith exactly",
+					filter: fmt.Sprintf(`"resource.uri".startsWith("%s")`, attestationOccurrence.Resource.Uri),
+					expected: []*grafeas_go_proto.Occurrence{
+						attestationOccurrence,
+					},
+				},
+				{
+					name:   "match kind startsWith partially",
+					filter: `"kind".startsWith("VULN")`,
+					expected: []*grafeas_go_proto.Occurrence{
+						vulnerabilityOccurrence,
+						secondVulnerabilityOccurrence,
+						alpineVulnerabilityOccurrence,
+						secondAlpineVulnerabilityOccurrence,
+					},
+				},
+				{
+					name:   "match resource.uri startsWith partially",
+					filter: `"resource.uri".startsWith("https://docker.io/library/alpine")`,
+					expected: []*grafeas_go_proto.Occurrence{
+						alpineVulnerabilityOccurrence,
+						secondAlpineVulnerabilityOccurrence,
+					},
+				},
+				{
+					name:     "match kind startsWith no match",
```

Comment:
this reads a little funky, can this wording be cleaned up?  maybe `match nothing via startsWith` or something like that